notify/impl.go

```diff
@@ -38,6 +39,25 @@ import (
 	"github.com/prometheus/alertmanager/types"
 )
 
+var (
+	numNotifications = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "alertmanager",
+		Name:      "notifications_total",
+		Help:      "The total number of attempted notifications.",
+	}, []string{"receiver", "integration"})
```

Comment:
I can imagine the number of receivers getting really high (thousands) in large companies, I wonder if we should just split out by integration and leave the rest to logs.